core/src/main/java/hudson/model/StreamBuildListener.java

```diff
@@ -80,5 +93,41 @@ public void finished(Result result) {
         getLogger().println("Finished: "+result);
     }
 
+    @Override
+    public void close() throws IOException {
```

Comment:
It implies that all classes are tolerant against douple close() call. They should according to the Closeable speck, so fine for me